src/main/java/com/ericsson/ei/queryservice/ProcessMissedNotification.java

```diff
@@ -84,40 +75,10 @@
 
     }
 
-    /**
-     * This method is responsible for fetching all the missed notifications from
-     * the missed Notification database and return it as JSONArray.
-     *
-     * @param request
-     * @param MissedNotificationDataBaseName
-     * @param MissedNotificationCollectionName
-     * @return JSONArray
-     */
-    public JSONArray processQueryMissedNotification(JsonNode request, String MissedNotificationDataBaseName, String MissedNotificationCollectionName) {
-        DB db = new MongoClient().getDB(MissedNotificationDataBaseName);
-        Jongo jongo = new Jongo(db);
-        MongoCollection aggObjects = jongo.getCollection(MissedNotificationCollectionName);
-        LOGGER.debug("Successfully connected to MissedNotification database");
-        MongoCursor<Document> allDocuments = aggObjects.find(request.toString()).as(Document.class);
-        LOGGER.debug("Number of document returned from Notification collection is : " + allDocuments.count());
-        JSONArray jsonArray = new JSONArray();
-        JSONObject doc = null;
-        while (allDocuments.hasNext()) {
-            Document temp = allDocuments.next();
-            try {
-                doc = new JSONObject(temp.toJson());
-            } catch (Exception e) {
-                LOGGER.error(e.getMessage(), e);
-            }
-            jsonArray.put(doc);
-        }
-        return jsonArray;
-    }
-
     @PostConstruct
     public void init() {
-        LOGGER.debug("The Aggregated Database is : " + missedNotificationDataBaseName);
-        LOGGER.debug("The Aggregated Collection is : " + missedNotificationCollectionName);
+        LOGGER.debug("MissedNotification Database is : " + missedNotificationDatabaseName);
+        LOGGER.debug("MissedNotification Collection is : " + missedNotificationCollectionName);
```

Comment:
Since EI runs in threads, the log message be spread out in the log. Therefore I prefer to to have these log messages in ono Log debug message. E.g LOGGER.debug(....Database is...... \n + ........Collection is: .......)